cookbooks/ambari/metadata.rb

```diff
@@ -0,0 +1,9 @@
+name 'ambari'
+maintainer 'Artem Ervits'
+maintainer_email 'generic13 at gmail dot com'
+license 'Apache'
+description 'Installs/Configures ambari-views-chef'
+long_description 'Installs/Configures ambari-views-chef'
+version '0.1.0'
+
+depends 'postgresql', '~> 6.1.1'
```

Comment:
Do we anticipate newer version to not work with this? If not, it is best to float with upstream so we know breakages sooner rather than later.